modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -507,18 +509,29 @@ inline IE::Blob::Ptr extractRemoteBlob(IECallContext& ctx, std::size_t i) {
     cv::util::any any_blob_params = ctx.inFrame(i).blobParams();
     auto ie_core = cv::gimpl::ie::wrap::getCore();
 
-    using ParamType = std::pair<InferenceEngine::TensorDesc,
-                                InferenceEngine::ParamMap>;
+    using ParamType = std::pair<std::pair<InferenceEngine::TensorDesc,
+                                 InferenceEngine::ParamMap>,
+                                 std::pair<InferenceEngine::TensorDesc,
+                                 InferenceEngine::ParamMap>>;
 
     ParamType* blob_params = cv::util::any_cast<ParamType>(&any_blob_params);
     if (blob_params == nullptr) {
         GAPI_Assert(false && "Incorrect type of blobParams: "
-                              "expected std::pair<InferenceEngine::TensorDesc,"
-                                                 "InferenceEngine::ParamMap>");
+                              "expected std::tuple<InferenceEngine::TensorDesc,"
+                                                  "InferenceEngine::ParamMap,"
+                                                  "InferenceEngine::TensorDesc,"
+                                                  "InferenceEngine::ParamMap>");
     }
 
-    return ctx.uu.rctx->CreateBlob(blob_params->first,
-                                   blob_params->second);
+    auto y_blob = ctx.uu.rctx->CreateBlob(blob_params->first.first, blob_params->first.second);
```

Comment:
I'd add a simple comment here, that the parameters are `TensorDesc` and `ParamMap`

Reply:
Done